src/style.css

```diff
@@ -1 +1,71 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(../src/fonts/Avenir-Book.ttf), url(../src/fonts/Avenir-Heavy.ttf);
+}
+
+.body {
+  margin: 0;
+  box-sizing: border-box;
+  padding-top: 20px;
+  font-family: Avenir, sans-serif;
```

Comment:
Remember to specify fallback fonts. While 'Avenir' might not be available on all systems, providing a generic font family like `sans-serif` is a good practice.